docs/workshop.md

```diff
@@ -555,15 +554,24 @@ The first Azure Function API created in the Lab offers a first security layer to
 [blob-output]: https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-output?tabs=python-v2%2Cin-process&pivots=programming-language-csharp
 [azure-managed-identity]: https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-output?tabs=python-v2%2Cisolated-process%2Cnodejs-v4&pivots=programming-language-csharp#identity-based-connections
 [azure-storage-extension]: https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestorage#:~:text=Installation.%20Download%20and%20install%20the%20Azure%20Storage%20extension%20for%20Visual
+[postman]: https://www.postman.com/
+
 ---
 
 # Lab 2 : Process the audio file with an Azure Durable Function
```

Comment:
fix Lab2 title markdown 

Tu m'avais parlé de ça mais je sais pas si tu l'as réglé, je vois rien sur le Lab 2 en particulier

Reply:
I have already fixed it, is the missing spaces before the separator `---` :-)